Jenkinsfile

```diff
@@ -43,6 +45,18 @@ timestamps {
     } // stage
 
     stage('Setup') {
+
+      // download NDK
```

Comment:
I'm already tying the build to OS X nodes anyways, so just download the darwin idk version, and unzip it into the working directory, then when we stash the workspace later it'll be included.